chia/daemon/client.py

```diff
@@ -70,19 +71,17 @@ async def _get(self, request: WsRpcMessage) -> WsRpcMessage:
             raise Exception("Websocket is not connected")
         asyncio.create_task(self.websocket.send_str(string))
 
-        async def timeout():
+        async def timeout() -> None:
             await asyncio.sleep(30)
             if request_id in self._request_dict:
-                print("Error, timeout.")
-                self._request_dict[request_id].set()
+                self.response_dict.pop(request_id)
```

Comment:
Shouldn't we still call `self._request_dict[request_id].set()` to signal the caller waiting on `wait()` below?

Reply:
yeah i'm already fixing it, it's what you mentioned earlier